federatedscope/autotune/algos.py

```diff
@@ -398,6 +401,27 @@ def _setup(self):
 
         return new_init_configs
 
+class FTSAlgo(Scheduler):
+    """FTS algorithm runs like a standard FL,
+        e.g. The server maintain a GP model for a client. At each
+        communication round, the server samples a config from the GP model
+         and broadcasts to the target client. A client training a moddel
+          from scratch on its local data with received hyper-param and return
+          the validation results to the server.
+    """
+
+    def _setup(self):
```

Comment:
We'd prefer to use `TrialExecutor` in https://github.com/alibaba/FederatedScope/blob/71bc828b4b6ad5d116de12c980b352500af314ac/federatedscope/autotune/algos.py#L38.

Besides, can we use FTS as a wrapper to wrap optimizers like FedEx here? You can refer to: https://github.com/alibaba/FederatedScope/blob/71bc828b4b6ad5d116de12c980b352500af314ac/federatedscope/autotune/algos.py#L322.

Reply:
I think FTS has a similar role as FedEx, what we want is to apply common HPO toolkit / our SHA to wrap FTS.